app.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env sh
+#
+# This script is run by OpenShift's s2i. Here we guarantee that we run desired
+# sub-command based on env-variables configuration.
+#
+
+case $THOTH_BUILD_ANALYSER_SUBCOMMAND in
+	'analyse')
+		exec /opt/app-root/bin/python3 thoth-build-analyser analyse
```

Comment:
Just a suggestion: we might want to change the name of the script in `setup.py` from `thoth-build-analysis` to `thoth-build-analyser` to be uniform in the naming as we're using that all over the place.

Reply:
Thats a really good suggestion, having consistency in the module is correct in my opinion.
i was thinking we should have consistency in the modules as well, i.e we have analyzer(in american english) and build-analysers(in british english).
It make sense to me to have `build-analysers` **->** `build-analyzer`

What do you think @fridex  @CermakM ?

Reply:
+1 on my side.

Reply:
+1